.github/workflows/regression-report.yml

```diff
@@ -398,6 +399,10 @@ jobs:
             needs.handle_existing_config_bundle.outputs.config_link
             || needs.handle_configs_in_pr_description.outputs.config_link
             }}
+          PATCH_CONFIG_LINK: ${{
+            needs.handle_configs_in_pr_description.outputs.patch_config_link
+            || ''
+            }}
           DIFF_PROJECTS_LINK: ${{
```

Comment:
Please add one more step `- name: Print patch config.xml`
One day it will help in debug of problem, beware it is not always defined

Reply:
done